examples/00_quick_start/embdotbias_movielens.ipynb

```diff
@@ -0,0 +1,1409 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<i>Copyright (c) Recommenders contributors.</i>\n",
+    "\n",
+    "<i>Licensed under the MIT License.</i>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## EmbeddingDotBias Recommender\n",
```

Comment:
@jmarrietar can you add the tag `parameters` in the parameter cell? See:
![image](https://github.com/user-attachments/assets/89b71060-8810-467d-ac93-d2cd136942f7)

This is used to inject parameters from the tests. That is the reason why your functional and smoke tests take the same time, because in the smoke test instead of using EPOCHS=1 it is using the default number of epochs.